src/DocBlock/Tags/See.php

```diff
@@ -56,6 +56,10 @@ public static function create(
         $parts       = preg_split('/\s+/Su', $body, 2);
         $description = isset($parts[1]) ? $descriptionFactory->create($parts[1], $context) : null;
 
+        if (preg_match('{^https?://}', $parts[0])) {
+            return null;
```

Comment:
@felixfbecker it's a good start that covers most cases I saw in the wild. I'm not against adding more but I can't promise I'll have time to do it.